infra/azure.tf

```diff
@@ -101,9 +101,11 @@ cat <<'CRON' > /root/daily-reset.sh
 set -euo pipefail
 echo "$(date -Is -u) start"
 
-AZURE_PAT=${secret_resource.vsts-token.value}
+echo "$(date -Is -u) arg: $1"
+target="$(/root/get-targets.sh $1)"
```

Comment:
this means that `/root/daily-reset.sh` only accepts `low` or `high`; my original approach assumed that we might call it with custom targets but if that's not the case this is fine

Reply:
This is an interesting shift in perspective from general software development to whatever what I do these days is called. I understand the impulse, but this is a script that's only ever called in this context right here on this one machine for this one purpose. If we're not passing other arguments here, we're not passing other arguments and the script only needs to work for the ones we are passing.

If, in the future, we need new configurations, we can add new entries to the `case` statement.

I actually considered merging the two files, too, as `get-versions.sh` is only ever called by `daily-reset.sh`, but since things were working at that point I left that for a future refactor.

Reply:
absolutely, I get that simple is best here, but as I'm not familiar enough with our Azure setup I worried that something somewhere, perhaps in a repo I don't know, on some Microsoft server or in a bash script on your own laptop, invoked `daily-reset.sh` in a way that would have broken